src/lib/pages/backoffice/Home/Home.js

```diff
@@ -31,14 +32,22 @@ export default class Home extends Component {
 
         <Grid.Row columns={2}>
           <Grid.Column>
+            <Divider hidden />
```

Comment:
thanks! does it look better with the white spaces ?

Reply:
Not much really :( I think the fact that the boxes do not have the same size is bothering me.
I would prefer to have boxes of a fixed size, even if empty, and not having them jumping around depending on how many rows are there...